src/sidebar/util/oauth-client.js

```diff
@@ -203,21 +197,22 @@ export default class OAuthClient {
    * Make an `application/x-www-form-urlencoded` POST request.
    *
    * @param {string} url
-   * @param {Object} data - Parameter dictionary
+   * @param {Record<string, string>} data - Parameter dictionary
    */
   _formPost(url, data) {
-    // The `fetch` API has native support for sending form data by setting
-    // the `body` option to a `FormData` instance. We are not using that here
-    // because our test environment has very limited `FormData` support and it
-    // is simpler just to format the data manually.
-    const formData = queryString.stringify(data);
+    const params = new URLSearchParams();
+    for (let [key, value] of Object.entries(data)) {
+      params.set(key, value);
+    }
+    params.sort();
```

Comment:
If this is a test affordance, would you mind adding a brief comment to that effect? Or otherwise comment why these need sorting? I think we may have covered this in the earlier, larger PR, but I've forgotten.

I'm glad that the complexity around `FormData` is no more.

Reply:
Done. This is indeed a test affordance, although the usual reason to sort parameters would be to improve cache hit rates by making URLs with the same query parameters always order them in the same way.